packages/app/src/vis-packs/core/complex/MappedComplexLineVis.tsx

```diff
@@ -31,7 +32,14 @@ function MappedComplexLineVis(props: Props) {
   return (
     <>
       {toolbarContainer &&
-        createPortal(<ComplexLineToolbar />, toolbarContainer)}
+        createPortal(
+          <ComplexLineToolbar
+            initialXScaleType={axisMapping[dimMapping.indexOf('x')]?.scaleType}
+            initialYScaleType={valueScaleType}
+            withAutoScale={dims.length > 1} // with 1D datasets, `baseArray` and `dataArray` are the same
```

Comment:
Why not keeping `disableAutoScale` ? `withAutoScale` makes me think that the button will disappear if set to `false`.